cli/utils/http.go

```diff
@@ -85,40 +67,40 @@ func HTTPPutJSON(urlPath, jsonPayload string) *http.Response {
 }
 
 func HTTPQuery(request *http.Request) *http.Response {
```

Comment:
Unless this is used outside of the package, HTTPQuery + CreateHTTPRequest and anything else only used in the package should be package private (lower case first letters).